docs/Stream_Transformation_Functions.md

```diff
@@ -0,0 +1,52 @@
+# Stream transformation functions
```

Comment:
Hi Anais, 
I'd recommend writing required guidelines as a step-by-step:
 "Create a transformation function" (...same for other functions, so they're consistent). Something like:

**Create a transformation function**

1. Conform to the examples in <link to universe> and include all [required functions and methods](#required-function-and-methods) in your transformation function.
2. Submit a unit test <insert link to folder>.
3. Submit end-to-end test. For details, see...
4. Add a description of the function to [SPEC.md](./docs/SPEC.md)